chapters/git-advanced.Rmd

```diff
@@ -1170,7 +1181,7 @@ they see that it contains all of the changes Amira made.
 knitr::include_graphics("figures/git-advanced/after-fork.png")
 ```
 
-A copy of a repository is called a \gref{clone}{git_clone}.\index{Git!clone a repository}\index{clone (in Git)}
+A copy of a repository is called a \gref{clone}{git_clone}(Figure \@ref(fig:git-cmdline-remote)c).\index{Git!clone a repository}\index{clone (in Git)}
```

Comment:
```suggestion
A copy of a repository is called a \gref{clone}{git_clone}.\index{Git!clone a repository}\index{clone (in Git)}
```
The new figure doesn't have `git clone` on it (which I think is a good thing), so the reference to the figure needs to be deleted here.

Reply:
`git clone` is included as a parenthetical for `git pull`. I'm happy removing the reference in image and text; it was included in the original image and I considered it marginally useful here, but I don't think it's absolutely necessary.

Reply:
Whoops, I totally missed `git clone` there in brackets next to `git pull` in the figure. I think maybe remove it since `git clone` isn't introduced in the chapter that the figure appears in at all?